rust/core-lib/src/editor.rs

```diff
@@ -424,7 +424,7 @@ impl Editor {
         let mut saved = None;
         for region in sel_regions {
             if !region.is_caret() {
-                let val = self.text.slice_to_string(region.min(), region.max());
+                let val = self.text.slice_to_string(region);
```

Comment:
this is cool. 😎

Reply:
It would be, but I'm unsure about something. What if the region is "reversed," which can happen for example by pressing shift-left-arrow?